apps/src/lab2/views/Lab2Wrapper.tsx

```diff
@@ -12,18 +12,15 @@ import classNames from 'classnames';
 import moduleStyles from './Lab2Wrapper.module.scss';
 import ErrorBoundary from '../ErrorBoundary';
 import {LabState, isLabLoading, hasPageError} from '../lab2Redux';
-import FontAwesome from '@cdo/apps/templates/FontAwesome';
-const i18n = require('@cdo/locale');
 
 import {ErrorFallbackPage, ErrorUI} from './ErrorFallbackPage';
 import Lab2Registry from '../Lab2Registry';
+import Loading from './Loading';
 
 export interface Lab2WrapperProps {
   children: React.ReactNode;
 }
 
-const noFade = window.location.href.includes('lab2-no-fade');
```

Comment:
@breville @sanchitmalhotra126 this didn't seem likely to be useful anymore so I removed it. Let me know if I should keep it.

Reply:
This was a recent addition in https://github.com/code-dot-org/code-dot-org/pull/56732 and is functionality we still want.

Reply:
whoops will add back!